reconcile/test/fixtures/openshift_perf_params/t1-params.yaml

```diff
@@ -0,0 +1,55 @@
+http_rates:
```

Comment:
I would add a comment explaining this is an internal representation from `build_template_params`

Reply:
nit: the fixture directory should be called as the integration